### ISSUE TITLE
yapf doesn't indent long line

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida_core#2114]
For example have a look a the following commit and failing test.

https://github.com/szoupanos/aiida_core/compare/ad24a32106c3...6159f50b30d9
https://travis-ci.org/szoupanos/aiida_core/jobs/446050100

_yapf_ doesn't correct the long line (and it should) and _prospector_ fails (and from the discussion that I had with @ltalirz, indentation and code formatting is the job of _yapf_ and not of _prospector_).

**Comment by dev-zero** [comment]
can reproduce, but I guess there is not much we can do about it.

The line in question could be rewritten as follows instead:
```python
_CREATE_DB_COMMAND = (
    'CREATE DATABASE "{}" OWNER "{}"'
    "ENCODING 'UTF8' LC_COLLATE='en_US.UTF-8' LC_CTYPE='en_US.UTF-8' TEMPLATE=template0")
```

**Comment by ltalirz** [comment]
@dev-zero Do we understand why, though?

**Comment by dev-zero** [comment]
@ltalirz yapf does not split strings by itself, most likely because the position to sensibly split a string depends on the content of the string, but there is an open feature request: https://github.com/google/yapf/issues/408

**Comment by ltalirz** [comment]
Thanks! We can keep this open, periodically checking the progress in yapf.

**Comment by giovannipizzi** [comment]
I'm closing this, as there is not much we can do, except telling developers what to do. 
For this, yesterday I added a guide in the wiki on how to use the pre-commit hooks [here](https://github.com/aiidateam/aiida_core/wiki/Contributors-technical-guide#how-to-run-the-pre-commit-hooks). I've now added an explanation of what to do in this case, and a link to this issue.
When/if yapf fixes this and we upgrade its version, this will be solved automatically.
